File: js/1154.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(x => parseInt(x))
cont = 0
aux = 0
while (1){
    n = lines.shift()
    if (n<0){
        console.log((aux/cont).toFixed(2) + '')
        break;
    }
    cont++
    aux+=n
}

File: c/1154.c
#include <stdio.h>

int main(){
    double n, aux, cont=0;

    while(1){
        scanf("%lf", &n);
        if (n<0){
            printf("%.2lf\n", aux/cont);
            break;
        }
        aux+=n;
        cont++;
    }
}